library/src/main/scala/g8.scala

```diff
@@ -110,9 +110,9 @@ object G8 {
     group.nativeGroup.setListener(new STErrorHandler)
     group.registerRenderer(renderer)
 
-    val attributes = resolved mapValues { attr =>
-      if (isBoolean(attr)) isTruthy(attr)
-      else attr
+    val attributes = resolved ++ resolved.collect {
+      case (k, v) if isBoolean(v) =>
+        companionField(k) -> isTruthy(v)
```

Comment:
@eed3si9n this commit introduces the companion field as discussed. unfortunately (or maybe not), `?` is not a valid ST field character, so that idea is definitely buried now.